src/test/java/org/waarp/openr66/dao/database/DBHostDAOIT.java

```diff
@@ -28,10 +28,11 @@
 
     public void runScript(String script) {
         try {
-            ScriptRunner runner = new ScriptRunner(con, false, true); 
+            ScriptRunner runner = new ScriptRunner(con, false, true);
             URL url = Thread.currentThread().getContextClassLoader().getResource(script);
             runner.runScript(new BufferedReader(new FileReader(url.getPath())));
         } catch (Exception e) {
+            System.err.println(e);
```

Comment:
Prefer to use a logger than System.err, even in tests (such that we can decide the verbosity by the level)

Reply:
yes, the new version with fail() is better !